### PR TITLE
chore: bump plugin versions

### PR DESCRIPTION
**Pull request by abzcoding**
# Description

Bump plugin version

### CODE REVIEW

**Comment by kylo252** [comment]
I added an update for telescope and luasnip that got released after you made the PR, and also for which-key. 

---

Side node: hopefully you didn't miss that there's a quick command to generate this table into a file 😅 
```console
$ make generate_plugins_sha
```

**Review by kylo252 — APPROVED** [review]

**Comment by abzcoding** [comment]
> I added an update for telescope and luasnip that got released after you made the PR, and also for which-key.
> 
> Side node: hopefully you didn't miss that there's a quick command to generate this table into a file 😅
> 
> ```
> $ make generate_plugins_sha
> ```

yep i'm using the same command

**Comment by abzcoding** [comment]
but apparently this is bugged after your commit
```
packer.nvim: Error running config for nvim-treesitter: .../.local/share/lunarvim/lvim/lua/lvim/core/treesitter.lua:79: attempt to call method 'get_default' (a nil value)
44
```

**Comment by kylo252** [comment]
> but apparently this is bugged after your commit
> 
> ```
> packer.nvim: Error running config for nvim-treesitter: .../.local/share/lunarvim/lvim/lua/lvim/core/treesitter.lua:79: attempt to call method 'get_default' (a nil value)
> 44
> ```

Damn! that's a pretty old line, I wonder how it even made it this far without errors… oh, it's probably because it was rarely, if ever, triggered. I've been wanting to skip setting up treesitter entirely in headless, so I'll fix that ASAP in another PR.

**Comment by abzcoding** [comment]
@kylo252 i think we should revert the `which-key` one
Screen Shot 2021-12-28 at 12 23 24 PM

File: lua/lvim/plugins.lua [hunk 1]
```diff
@@ -29,10 +29,10 @@ local commit = {
   popup = "b7404d35d5d3548a82149238289fa71f7f6de4ac",
   project = "71d0e23dcfc43cfd6bb2a97dc5a7de1ab47a6538",
   structlog = "6f1403a192791ff1fa7ac845a73de9e860f781f1",
-  telescope = "9aaaa0c5f3eb665b51bbcafda084de4b0952fef0",
+  telescope = "88437804e157196f053d0fa62dc891facd9ab746",
   telescope_fzf_native = "b8662b076175e75e6497c59f3e2799b879d7b954",
   toggleterm = "265bbff68fbb8b2a5fb011272ec469850254ec9f",
-  which_key = "0fd9de78fe09215e1b7c6173ff1b0b90c8ed6ec4",
+  which_key = "312c386ee0eafc925c27869d2be9c11ebdb807eb",
```
Comment (abzcoding) [inline lua/lvim/plugins.lua:35]: @kylo252 this is causing errors
Screen Shot 2021-12-28 at 12 23 24 PM

**Comment by abzcoding** [comment]
nvm you can fix the which-key error by doing a `LvimCacheReset`

File: lua/lvim/plugins.lua (hunk 1 above)
Comment (abzcoding) [reply lua/lvim/plugins.lua:35]: nvm you can fix the which-key error by doing a `LvimCacheReset`

**Comment by kylo252** [comment]
> nvm you can fix the which-key error by doing a `LvimCacheReset`

close one I guess 😅 

I seem to recall it's some problem with impatient, but that should hopefully be fine if you pull this with `:LvimUpdate`.

**Pull request merged**